### PR TITLE
feat:Edit ecosystem modal component

### PR DESCRIPTION
**Pull request by pranalidhanavade**
## What ##

- Added edit ecosystem popup-modal component.
- Added Dropdown on ecosystem dashboard.
- Resolved CSS issue of error message.

### CODE REVIEW

File: src/components/Ecosystem/Dashboard.tsx [hunk 1]
```diff
@@ -32,28 +34,37 @@ const Dashboard = () => {
...
 	};
 
+	const EditEcosystemOrgModal = () => {
+		setEditOpenModal(true);
+	};
+
+	const handleEditModalClose = () => {
+		setEditOpenModal(false);
+		setDropdownOpen(false); // Close the dropdown when the edit modal is closed
+	  };
```
Comment (16-karan) [inline src/components/Ecosystem/Dashboard.tsx:60]: Remove the commented code

**Review by 16-karan — COMMENTED** [review]
Please resolve the commits

File: src/components/Ecosystem/Dashboard.tsx (hunk 1 above)
Comment (pranalidhanavade) [reply src/components/Ecosystem/Dashboard.tsx:60]: Resolved

File: src/components/EditEcosystemOrgModal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+import type { Ecosystem } from "../Ecosystem/interfaces";
+import React, { useEffect, useState } from "react";
+
+interface EditEntityModalProps {
+    openModal: boolean;
+    setMessage: (message: string) => void;
+    setOpenModal: (flag: boolean) => void;
+    onEditSuccess?: () => void;
+    entityData: Organisation | Ecosystem | null;
+    isOrganization: boolean; // Indicates whether it's an organization or ecosystem
```
[the remark this answers is not in the document]
Comment (pranalidhanavade) [reply src/components/EditEcosystemOrgModal/index.tsx]: Done

File: src/components/EditEcosystemOrgModal/index.tsx [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+                logoFile: "",
+                imagePreviewUrl: "",
+                fileName: "",
+            });
+            setImgError('');
+            setErrMsg(null);
+            setLoading(false);
+        }
+    }, [props.openModal]);
+    // Function to process image
```
[the remark this answers is not in the document]
Comment (pranalidhanavade) [reply src/components/EditEcosystemOrgModal/index.tsx]: Done

File: src/components/EditEcosystemOrgModal/index.tsx [hunk 4]
```diff
@@ -0,0 +1,340 @@
...
+                            fileName: file.name
+                        });
+                    }
+                };
+            }
+        };
+    };
+
+    // Function to handle image change
+    const handleImageChange = (event: any): void => {
```
[the remark this answers is not in the document]
Comment (pranalidhanavade) [reply src/components/EditEcosystemOrgModal/index.tsx:125]: Done

**Review by sanjay-k1910 — APPROVED** [review]
Looks good to me.

**Pull request merged**